test/Debugger/rlexe.xml

```diff
@@ -16,6 +16,8 @@
     <default>
       <compile-flags>-debuglaunch -dbgbaseline:JsDiagGetFunctionPosition.js.dbg.baseline</compile-flags>
       <files>JsDiagGetFunctionPosition.js</files>
+      <!-- xplat-todo: enable back [not supported] -->
```

Comment:
>not supported [](start = 36, length = 13)

Why? JsDiagGetFunctionPosition doesn't seems to have any xplat specific code? and it is used in node-chakracore debugging which seems to be working fine on xplat.